src/open_inwoner/accounts/urls.py

```diff
@@ -101,6 +102,7 @@
     path("contacts/", ContactListView.as_view(), name="contact_list"),
     path("onderwerpen/", MyCategoriesView.as_view(), name="my_categories"),
     path("mydata/", MyDataView.as_view(), name="my_data"),
+    path("notifications/", MyNotificationsView.as_view(), name="my_notifications"),
```

Comment:
I just like to note this 'My' naming pattern is awful but I'll accept it as at least it keeps the pattern.